deepinv/physics/forward.py

```diff
@@ -553,6 +622,23 @@ class Denoising(DecomposablePhysics):
     The linear operator is just the identity mapping :math:`A(x)=x`
 
     :param torch.nn.Module noise: noise distribution, e.g., ``deepinv.physics.GaussianNoise``, or a user-defined torch.nn.Module.
+
+    |sep|
+
+    :Examples:
+
+        Denoising operator with Gaussian noise with standard deviation 0.1:
+
+        >>> from deepinv.physics import Denoising, GaussianNoise
+        >>> seed = torch.manual_seed(0) # Random seed for reproducibility
+        >>> x = torch.randn(1, 1, 3, 3) # Define random 3x3 image
```

Comment:
```suggestion
        >>> x = 0.5*torch.ones((1, 1, 3, 3)) # Define random 3x3 image
```
(like that we will visually see that the result is noisy with mean ~0.5 whereas now since x is pure noise as well the effect of physics is not clear)